src/macros.rs

```diff
@@ -54,68 +384,104 @@ bitvec![1; 5];
 #[cfg(feature = "alloc")]
 #[macro_export]
 macro_rules! bitvec {
-	//  bitvec![ endian , type ; 0 , 1 , … ]
-	( $order:path , $bits:ty ; $( $val:expr ),* ) => {
-		bitvec![ __bv_impl__ $order , $bits ; $( $val ),* ]
+	($order:ident, $bits:ident; $($val:expr),* $(,)?) => {
```

Comment:
This should probably have cases for `Local`, like `bits!` does.

Alternatively, all of the `bitvec![1, 2, 3]` variants could call `bits!` under the hood, as they don't need a dynamic length (which is the only thing `bits!` doesn't support).

Reply:
Yeah, once I think `bits!` is stabilized enough I want `bitvec!` to forward to it and just run `Self::from_bitslice` on the output. Reduces argparse code and forces me to ensure that the library implementation is actually good.